server/src/main/java/org/tbbtalent/server/repository/db/CandidateSpecification.java

```diff
@@ -286,7 +286,7 @@ public static Specification<Candidate> buildSearchQuery(
             }
 
             // REFERRER
-            if (request.getRegoReferrerParam() != null) {
+            if (request.getRegoReferrerParam() != null && !request.getRegoReferrerParam().equals("")) {
```

Comment:
Even better would be !request.getRegoReferrerParam().trim().equals("")

That will work if there are all spaces - which will also be invisible to user

Reply:
Or maybe simpler request.getRegoReferrerParam().trim().length() != 0